internal/pkg/earpointer/analysis_test.go

```diff
@@ -65,9 +77,15 @@ func TestFieldAddr(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	want := "{*f.a}: [x->f.t0], {f.a}: --> *f.a, {f.b}: [], {f.t0}: --> f.b"
-	if got := state.String(); got != want {
-		t.Errorf("got: %s\n want: %s", got, want)
+	want := map[string]string{
```

Comment:
`want := partitionItemsToString(map[string]string{...})`

This gets rid of the pesky "wantstr" name.